site/conf.py

```diff
@@ -652,7 +652,7 @@
 # Where the output site should be located
 # If you don't use an absolute path, it will be considered as relative
 # to the location of conf.py
-# OUTPUT_FOLDER = 'output'
+# OUTPUT_FOLDER = '../blog'
```

Comment:
Why this change? What has to do with the tag cloud?

Reply:
Nop.
I was just to put it back on default commented config .